### DISCUSSION TITLE
Auth 404

### DISCUSSION THREAD

**Discussion by need4swede** [DIGITALCRIMINALS/OnlyFans#322] [Q&A]
Keep getting:

Auth (V1) Attempt x/10
Auth 404'ed

Eventually ends with:

apis/onlyfans/classes/auth_model.py", line 223, in get_user
    response["session_manager"] = self.session_manager
TypeError: 'NoneType' object does not support item assignment

So I tried running the tool again, with a new browser session, and now I'm getting:

main_helper.py", line 976, in process_profiles
    json_auth = temp_json_auth["auth"]
KeyError: 'auth'

Is this happening to anyone else?

**Comment by need4swede** [comment]
I noticed that my username keeps changing to 'default' in the json; is this by design?

**Comment by SterlingSavariau** [comment]
I'm getting the same horseshit error

**Comment by DIGITALCRIMINAL** [comment]
Does your auth.json contain a "auth" key? It should look similar to the one on the readme.

**Reply by SterlingSavariau** [reply]
Yes. Exactly similar.
